### DISCUSSION TITLE
`pinyin` support

### DISCUSSION THREAD

**Discussion by catundercar** [meilisearch/product#204] [Feedback & Feature Proposal]
need `pinyin` participle。 like `温湿度`  can get the Chinese alphabet and abbreviation `wsd` , `wenshidu`...

**Comment by curquiza** [comment]
Hello here!

We are currently working (with @ManyTheFish) on the [Meilisearch's tokenizer](https://github.com/meilisearch/charabia) and we are considering any possibility to improve the Chinese language support. However, no one at Meilisearch speaks Chinese, which means we need the community point of view 🙌

Here are the current issues we have issues with Chinese language support (I don't say this could be fixed with pinyin):
- https://github.com/meilisearch/meilisearch/issues/2222
- https://github.com/meilisearch/meilisearch/issues/2091

**For any people who frequently use pinyin, could you help us identify the pros and the cons of using it?**

@ManyTheFish feel free to ask or add anything that could be useful!
